maingateway-service/README.md

```diff
@@ -0,0 +1,69 @@
+# API Gateway Service
+
+
+This demo demonstrates how to use SQL via JDBC along with Camel's REST DSL to expose a RESTful API.
+
+This example relies on the Fabric8 Maven plugin for its build configuration and uses the FIS Java base image
+
+
+
+## Background
+
+
+
```

Comment:
'uses SQL'  instead of 'use SQl' ?
connects -> connect ?
'reterieve' -> retrieves ?
'By' -> by ?
'jason' -> 'Jason'

Reply:
json

Reply:
should be to all cap? JSON or just json?

Reply:
Sure, JSON would be better. Dropping the "a" is most important though :-)

Reply:
done .

Reply:
Whoever first makes the comment (in these cases, Marco), should review and resolve the conversation when satisfied.